source/codegen/metadata/nidcpower/config.py

```diff
@@ -1,29 +1,166 @@
 # -*- coding: utf-8 -*-
-# This file is generated from NI-DCPower API metadata version 20.7.0d8
+# This file is generated from NI-DCPower API metadata version 23.0.0d9999
```

Comment:
Await: I bumped this from an in flight AzDo PR: https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/376469

Will bump from `nidcpower_grpc_device` export after that PR goes in and resolve this comment.

Reply:
PR went through and now imported metadata, proto, headers, and .lib files come from `nidcpower_grpc_device` export 23.0.0d114.